### PR TITLE
Updated Introduction by simplifying the writing

### PR DESCRIPTION
**Pull request by ijlee2**
## Description

I was reviewing what we mention in the CLI Guides with regards to the folder structure for an in-repo addon. On the long horizon, I'd like to document the colocated folder structure in Octane (this pertains to the issue https://github.com/ember-learn/cli-guides/issues/135).

As I review each section, if it's okay with the team, I'd like to update the section by simplifying the writing (the grammar). For example, in this PR, I preferred 2 short sentences instead of 1 joined by an "and" or a "which." I'm hoping that the result is a writing that is easier to parse and feels conversational when read out aloud.

### CODE REVIEW

File: guides/index.md [hunk 1]
```diff
@@ -11,37 +11,43 @@ for information on the latest releases and new features.
...
 
-Although these CLI guides have a lot of content, a typical Ember developer will only need to know and use a small fraction of the information.
-For new users, the recommended learning path is to first do the Ember.js [Quickstart](https://guides.emberjs.com/release/getting-started/quick-start/) and [Tutorial](https://guides.emberjs.com/release/tutorial/), which teach the commands while building a sample app. Then, review the Basic Use section of this site as a reference resource.
+You will find a lot of information in the Ember CLI Guides. Rest assured, you will only need to know and use a small fraction in a typical day.
 
-The CLI comes with a command-line-based help system too. At any point, if you're curious what a command does and what the options are, you can add `--help` to the command and press enter to find out more.
+For new users, the recommended learning path is to first do the Ember.js [Quickstart](https://guides.emberjs.com/release/getting-started/quick-start/) and [Tutorial](https://guides.emberjs.com/release/tutorial/). These teach the commands while building a sample app. Then, review the [Basic use](./basic-use) section of this site as a reference.
```
Comment (ijlee2) [inline guides/index.md:16]: I added a link to the `Basic use` section.

File: guides/index.md [hunk 2]
```diff
@@ -11,37 +11,43 @@ for information on the latest releases and new features.
...
+
+The CLI comes with a help system too. At any point, if you're not sure what a command does and what its options are, you can add `--help` to the command and press enter to find out more.
+
+```shell
```
Comment (ijlee2) [inline guides/index.md:20]: I added an example of using `--help`.

File: guides/index.md [hunk 3]
```diff
@@ -11,37 +11,43 @@ for information on the latest releases and new features.
...
-you can install and import it in your application without any other configuration. Be sure to visit the [ember-auto-import](https://github.com/ef4/ember-auto-import) documentation for more advanced usage!
+Since `v3.x`, Ember apps include the `ember-auto-import` dependency, which enables importing npm packages directly. For example, if you want to use `highcharts` in your application,
+you can install and import it without any other configuration. Be sure to visit the [ember-auto-import](https://github.com/ef4/ember-auto-import) documentation for more advanced usage!
 
 ## Why do we need a CLI?
 
-The Ember CLI is like a dependency packager, test runner, optimizer, and local server all rolled into one. Since all the features were built to work together, common tasks (such as upgrading the app version or deploying) can be automated with production-ready, open source plugins. The CLI is backwards-compatible with older Ember apps and maintains a six-week release schedule.
+The Ember CLI's job is to make your work easier.
```
Comment (ijlee2) [inline guides/index.md:37]: I liked this sentence so I moved it to the top of the subsection (previously, line 33).

File: guides/index.md [hunk 4]
```diff
@@ -11,37 +11,43 @@ for information on the latest releases and new features.
...
 
-However, if you want to make your own adjustments like using a different testing library, you still can.
+The CLI was also built with the idea that a developer should be able to focus on building great apps, not re-engineering how to fit pieces together throughout an app's lifecycle. Apps become maintainable and approachable, since there are established architectural patterns across individuals, teams, and companies.
+However, if you want to make your own adjustments, such as using a different testing library, you still can.
 
 ## Getting help
 
-Do you have questions? Run into an issue or a bug? Get support from the community. A list of chat rooms, forums, and more are available [here](https://www.emberjs.com/community/).
+Did you have questions? Run into an issue or a bug? Get support from the community. A list of chat rooms, forums, and more is available on the [Community](https://www.emberjs.com/community/) page.
```
Comment (ijlee2) [inline guides/index.md:46]: I changed "are" to "is" and provided a descriptive information for the link ("here" is, in general, discouraged, I believe).

File: guides/index.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
-The Ember CLI (command line interface) is the official way to create, build, test, and serve the files that make up an Ember.js app or addon.
-Many things have to happen before any web app is ready for the browser, and the Ember CLI helps you get there with zero configuration.
+The Ember CLI (command line interface) is the official way to create, build, test, and serve the files that make up an Ember app or addon.
+Many things have to happen before a web app is ready for the browser. The Ember CLI helps you get there with zero configuration.
```
Comment (rwjblue) [inline guides/index.md:2]: I think generally "the Ember CLI" is a bit odd. I'd personally say:

```suggestion
Many things have to happen before a web app is ready for the browser. Ember CLI helps you get there with zero configuration.
```

Comment (ijlee2) [reply guides/index.md:2]: Yep, I think removing the article sounds better. Thanks for the suggestion!

File: guides/index.md (hunk 3 above)
Comment (ijlee2) [reply guides/index.md:37]:
```suggestion
Ember CLI's job is to make your work easier.
```

**Pull request merged**